src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -655,4 +656,15 @@ BOOST_AUTO_TEST_CASE(MishFunctionTest)
   CheckDerivativeCorrect<MishFunction>(desiredActivations,
                                         desiredDerivatives);
 }
+BOOST_AUTO_TEST_CASE(SoftShrinkageTest)
+{
+  const arma::colvec desiredActivations("-1.5 2.7 4 -99.7 0.5 -0.5 1.5 0");
+  
+  const arma::colvec desiredDerivatives("1 1 1 1 1 1 1 0");
+
+  CheckActivationCorrect<SoftShrinkage>(activationData,
+                                       desiredActivations);
+  CheckDerivativeCorrect<SoftShrinkage>(desiredActivations,
+                                        desiredDerivatives);                                          
```

Comment:
This line is more than 80 chars long. Could you fix that, remove whitespace after ;

Reply:
Could you remove space after `;` ,
Thanks.